src/Server.h

```diff
@@ -70,6 +70,14 @@ class cServer
 	size_t GetNumPlayers(void) const { return m_PlayerCount; }
 	void SetMaxPlayers(size_t a_MaxPlayers) { m_MaxPlayers = a_MaxPlayers; }
 
+	// tolua_end
+	/** Add a Forge mod to the server ping list. */
+	bool RegisterForgeMod(AString & a_ModName, AString & a_ModVersion, UInt32 a_ProtocolVersionNumber);
+	// tolua_begin
```

Comment:
It's needed to complete the type for accessing m_Client in cProtocol_1_10_0::HandlePacketStatusRequest:

```c++
	m_Client->ForgeAugmentServerListPing(ResponseValue); 
```

but, I could move it to Protocol_1_10.cpp closer to where it is needed